components/authz-service/storage/v2/postgres/postgres_test.go

```diff
@@ -1327,12 +1327,11 @@ func TestCreatePolicy(t *testing.T) {
 				Actions:   actions0,
 			}
 			member0 := genMember(t, "user:local:albertine")
-			member1 := genMember(t, "user:local:albertine")
 
 			pol := storage.Policy{
 				ID:         polID,
 				Name:       name,
-				Members:    []storage.Member{member0, member1},
+				Members:    []storage.Member{member0, member0},
```

Comment:
Why duplicate the member?

Reply:
Before, this was member0 and member1, which both sharing a name, but having a different ID. Since there are no IDs anymore, this should be approximating the same test case.